API.md

```diff
@@ -13,9 +13,15 @@
 <dt><a href="#getViewportSize">getViewportSize(deviceName)</a> ⇒ <code><a href="#DeviceViewportSize">DeviceViewportSize</a></code></dt>
 <dd><p>Gets the viewport size for the specified device.</p>
 </dd>
+<dt><a href="#isMaximized">async isMaximized(pageTitle)</a> ⇒ <code>boolean</code></dt>
+<dd><p>Checks if the specified browser window is maximized.</p>
+</dd>
 <dt><a href="#isValidDeviceName">isValidDeviceName(inputString)</a> ⇒ <code>boolean</code></dt>
 <dd><p>Checks if the provided string is a valid device name contained in the screen size database.</p>
 </dd>
+<dt><a href="#maximize">async maximize(pageTitle)</a></dt>
```

Comment:
Maybe it's better to add `restoreDown` (or `unmaximize`) function? As I understand we have to use the following action chain because of https://github.com/DevExpress/testcafe-browser-natives/issues/71:
```
check-maximized
if maximized -> resize
resize
```
Maybe the following looks better:
```
check-maximized
if maximized -> restore down
resize
```
@AndreyBelym what do you think?